static/webui/script.js

```
@@ -898,10 +898,19 @@ function setFingerprint(ItemType, ItemId, fingerprint) {
 
 function doAcquireMod() {
     const uniqueName = getKey(document.getElementById("mod-to-acquire"));
+    const modCount = parseInt(document.getElementById("mod-count").value, 10);
+
     if (!uniqueName) {
         $("#mod-to-acquire").addClass("is-invalid").focus();
         return;
     }
+
+    if (isNaN(modCount) || modCount < 1) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)